components/dm/command/deploy.go

```diff
@@ -44,7 +43,7 @@ func newDeploy() *cobra.Command {
 			}
 
 			// natvie ssh has it's own logic to find the default identity_file
-			if gOpt.NativeSSH && !utils.IsFlagSetByUser(cmd.Flags(), "identity_file") {
+			if gOpt.NativeSSH && !tiuputils.IsFlagSetByUser(cmd.Flags(), "identity_file") {
```

Comment:
Please keep consistent with other places, https://github.com/pingcap/tiup/pull/699/files#diff-0303fd0be0010c69e4be3e5473419be0R103.
I think it's better to don't use the `gOpt.NativeSSH` at the rest of the code, it should be only present at `root.go` to handle compatibility.